### DISCUSSION TITLE
Development Documentation

### DISCUSSION THREAD

**Discussion by Gerardlc** [unopim/unopim#81] [Q&A] [answered]
In order to improve the project and be able to contribute developments, it would be very good to have a development guide.

For example, the project says that it uses Vuejs for the frontend and I don't see a single .vue file in the project, where are they? How is the frontend part developed?

Is all the code in the "packages/Webkul" folder?

**Comment by devansh-pal-webkul** [comment, marked as answer]
@Gerardlc,
To better understand how to contribute to the project, we suggest referring to the [development documentation](https://devdocs.unopim.com/). This should provide a comprehensive guide on the project structure, how to work with the frontend, and other important development practices.

Also you don't see any .vue files because we are using vue as embedded components inside of a blade file, so technically all files are blade templates but can have vue components inside of them.
